superset-frontend/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -87,6 +87,8 @@ const DropdownButton = styled.div`
   margin-left: ${({ theme }) => theme.gridUnit * 2.5}px;
 `;
 
+const screenshotNodeSelector = '.dashboard';
```

Comment:
super nit: since this is a constant, prefer SCREENSHOT_NODE_SELECTOR

Reply:
Done